### DISCUSSION TITLE
Is noise floor receive sensitivity?

### DISCUSSION THREAD

**Discussion by bukepo** [openthread/openthread#10803] [Q&A]
IIUC, receive sensitivity is static value defined in IEEE 802.15.4 and measured in lab, while noise floor should be runtime value measured in real environment. Currently the link margin is calculated based on sensitivity instead of noise floor as defined in Thread spec. Anything prevents calculating link margin based on noise floor?

**Comment by jwhui** [comment]
Thread Specification defines Link Margin as follows:
```
The Link Margin SHALL be calculated as follows:

    Link Margin = RSSI - noise_floor

where noise_floor is 1 of the following:
    • measured noise floor
OR
    • receiver sensitivity
OR
    • constant
```

Nothing prohibits using noise floor.

**Reply by bukepo** [reply]
I see. However, it seems there's no interface for the platform to provide the measured noise floor in OpenThread. I also checked some vendor SDKs, but didn't find existing API to get the measured noise floor or SNR based on measured noise floor. I'll close this topic.
